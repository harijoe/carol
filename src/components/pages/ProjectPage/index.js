import React from 'react'
import PropTypes from 'prop-types'

import { MainLayout, MainWrapper, InnerWrapper } from 'components'
import { ProjectDetails } from 'containers'

const ProjectPage = props =>
  <MainLayout>
    <MainWrapper>
      <InnerWrapper>
        <ProjectDetails id={props.params.projectId} />
      </InnerWrapper>
    </MainWrapper>
  </MainLayout>

ProjectPage.propTypes = {
  params: PropTypes.shape({
    projectId: PropTypes.string,
  }),
}

export default ProjectPage
