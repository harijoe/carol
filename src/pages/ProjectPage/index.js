import React from 'react'
import PropTypes from 'prop-types'

import MainLayout from 'layouts/MainLayout'
import MainWrapper from 'components/MainWrapper'
import ProjectDetails from 'containers/ProjectDetails'

const ProjectPage = props =>
  <MainLayout>
    <MainWrapper paddingTop="m">
      <ProjectDetails id={props.params.projectId} />
    </MainWrapper>
  </MainLayout>

ProjectPage.propTypes = {
  params: PropTypes.shape({
    projectId: PropTypes.string,
  }),
}

export default ProjectPage
