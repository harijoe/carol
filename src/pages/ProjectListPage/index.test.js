import React from 'react'
import { shallow } from 'enzyme'
import ProjectPage from './'

it('renders', () => {
  const wrapper = shallow(<ProjectPage />)

  expect(wrapper).toMatchSnapshot()
})
