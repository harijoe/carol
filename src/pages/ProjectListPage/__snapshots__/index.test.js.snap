// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<Layout>
  <MainWrapper
    paddingTop="m"
  >
    <Connect(ProjectsContainer) />
  </MainWrapper>
</Layout>
`;
